src/Options.php

```diff
@@ -150,7 +150,7 @@ class Options
     // Thumbnail Images Options
     private bool $writeThumbnail = false;
     private bool $writeAllThumbnails = false;
-    private string $convertThumbnail ;
+    private string $convertThumbnail = null ;
```

Comment:
I don't understand why you add a space after null, you need to fix CS

Reply:
Sorry, this is my first time to contribute something and environment is new for me :)

Reply:
You can run locally `vendor/bin/php-cs-fixer fix` :)

Reply:
I ran the php-cs-fixer with fix switch but still error here! PHP version is matter?